### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by dieum512**
In this branch we :

- Created a new branch

- Checked: 
1. Page tittles: we updated the page tittle
2. Image text alternatives: we added text to the alt
3. Text headings : we updated the text headings
4. Resize: we added a hover effect to some texts to increase the font size on hover
5. interaction: No issue found
6. Moving content: No moving content
7. Multimedia: No multimedia content

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -304,7 +304,7 @@ <h2>Skills</h2>
     <section class="contact-section">
       <div class="contact-container">
         <div class="contact-div">
-          <h1>Contact me</h1>
+          <h2>Contact me</h2>
```
Comment (anagudelogu) [inline index.html:305]: - [ ] Great job working on the headings of your page! 💯 In order to improve further the accessibility of your page you could modify the labels of your contact form as explained below.

Following the [Easy Checks](https://www.w3.org/WAI/test-evaluate/preliminary/#forms) accessibility instructions for forms:

image

Please add an indicator of which fields are required! 🤓 🛠️

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi Team! @dieum512 @Donmark2k, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 

## Required Changes ♻️

- [ ] According to the requirements, your PR description should contain a report listing all points that you have checked, and the result of your evaluation. 
image
Please consider editing your pull request description in order to meet this checklist! 🤓 🛠️ 

- [x] I noticed that you didn't change your page title, at the moment, `"portfolio"` is too generic, titles should adequately and briefly describe the content of the page. For example:

* Poor titles:
  * *Microverse project | Your portfolio.*
  * *Portifolio project*
* Better page titles:
  * *Mark's portifolio*
  * *Mark full-stack webdeveloper*

Please consider applying this suggestion! 🤓 🛠️ 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🚧 

Hi Team! @dieum512 @Donmark2k 👋🏼 
I'm Andres.

You've done a great job implementing some of the requested changes for this milestone 🚀, but there are still some which haven't been addressed yet. You're getting closer, keep up the good work! 🤖 🎆 



 ## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

 ## Required Changes ♻️

- [ ] You guys did a great job following the advice with the portfolio title! 💯 However, In order to meet the previous review requirements and improve the accessibility of your page, please modify the labels of the required fields of your contact form as explained below.

Following the [Easy Checks](https://www.w3.org/WAI/test-evaluate/preliminary/#forms) accessibility instructions for forms:

image

https://github.com/dieum512/Portfolio/blob/cb9499396c4cc477c7dfc1cdcb8bade7e32ed398/index.html#L311-L344

Please add an indicator of which fields are required (By convention, you could add an asterisk `*` to the label/placeholder text to indicate that it is required)! 🤓 🛠️ 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @dieum512 
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs





 
 ## Optional suggestions
 
 - Please check the comment under the review.
 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**